extensions/cep/artifacts/wso2cep-3.0.0/eventformatters/AverageInFlightRequestsWSO2EventFormatter.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2005-2015 WSO2, Inc. (http://wso2.com)
+  ~
```

Comment:
Thanks for pointing this. These files were in cep extension for ppaas 4.1.0 release and we no longer have the cep extension in private paas repo. It's there in stratos repo.